internal/core/command/init.go

```diff
@@ -143,6 +143,8 @@ func initializeConfiguration(useRegistry bool, useProfile string) (*Configuratio
 		}
 	}
 
+	config.OverwriteFromEnvironment(configuration)
```

Comment:
I admit I haven't looked at this really closely -- been taken up with other things. What I have looked at doesn't show a way to toggle this behavior. Maybe just point me in the right direction. We definitely don't want to do this by default.

Reply:
The behavior can't be toggled per-se.  

If you set an environment variable named after a valid configuration field, the variable's value will overwrite the value already in configuration.  If you don't set appropriately named environment variables, the configuration will not be overridden.

Not sure if the concern is overhead to run the function.  It runs once during start up and time consumption is negligible.